### ISSUE TITLE
Colorize or website logo

### ISSUE THREAD

**Issue by Lechucico** [mattrubin/authenticator#249]
There should be an option to edit the color of the account or at least the logo of the website appears next to the name. That would be much easier and readable.

**Comment by DidierLmn** [comment]
Indeed, website icons would make it significantly easier to locate the right entry when you have a lot of them. This would be a great feature.

**Comment by ohreally** [comment]
Adding website icons would probably require a network connection. One of the important reasons for me to use this specific tool, is that it is not connected.
Felt I should mention it...

User configurable background colours/shades would be quite an improvement, though.
